### PR TITLE
Display a message instead of landing page if timeline is empty

### PR DESCRIPTION
**Pull request by cedricmessiant**
Fixes DEVAS-871

### CODE REVIEW

File: assembl/static2/js/app/components/common/messagePage.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+type Props = {
+  title: ?string,
+  text: string
+};
+
+const MessagePage = ({ title, text }: Props) => (
+  <Grid fluid>
+    <div className="max-container">
+      <Jumbotron>
+        {title && <h3>{title}</h3>}
```
Comment (MilanBarande) [inline assembl/static2/js/app/components/common/messagePage.jsx]: Just an h3, no styleguide class ?

File: assembl/static2/js/app/utils/translations.js [hunk 2]
```diff
@@ -560,7 +561,8 @@ const Translations = {
       contact: "Do you wish to know more about this platform?",
       contactUs: "Contact us",
       from_start_to_end: "from %{start} to %{end}",
-      chatbot: "Chat with %{chatbotName}"
+      chatbot: "Chat with %{chatbotName}",
+      assemblNotConfigured: "Assembl is not configured properly"
```
Comment (MilanBarande) [inline assembl/static2/js/app/utils/translations.js]: I wouldn't say "not configured properly" but rather "has not been fully configured yet"

File: assembl/static2/js/app/components/common/messagePage.jsx (hunk 1 above)
Comment (cedricmessiant) [reply assembl/static2/js/app/components/common/messagePage.jsx]: done, thanks

File: assembl/static2/js/app/utils/translations.js (hunk 2 above)
Comment (cedricmessiant) [reply assembl/static2/js/app/utils/translations.js]: done

**Pull request merged**